### DISCUSSION TITLE
Unexpected behaviour with colours

### DISCUSSION THREAD

**Discussion by imprisonedmind** [tailwindlabs/tailwindcss#7488] [Help] [answered]
Hi there, Not sure if this is a bug or we are doing something wrong. 

The project is using Next Js and Tailwind CSS. 

Expected behaviour: bg-orange-200 produces class with correct properties. 
Current behaviour: bg-orange-200 produces an inline class but it has no effect on the element. 

This behaviour happens with multiple colours. Most seem to only work on -500 or -100 but text colours are also breaks.

**State Comp:** 
```
import React from "react"
import {StateMap} from "../../lib/state-map";
import PropTypes from "prop-types";
import Text from "../Text/Text";

const State = ({state, size}) => {
    let displayValue = StateMap[0].display
    let colours = StateMap[0].colours

    if (StateMap[state]) {
        displayValue = StateMap[state].display
        colours = StateMap[state].colours
    }

    if (size === "large") return <div className={`flex h-6 md:h-8 my-auto px-4 rounded-full text-xl ${colours}`}>
        <Text size="default" style={'m-auto leading-6'} value={displayValue}/>
    </div>

    return <div className={`px-4 py-1 inline-block text-xs leading-none select-none rounded-full ${colours}`}>
        {displayValue}
    </div>
}

State.propTypes = {
    state: PropTypes.any.isRequired
}

export default State

```

**State Map**
```

export const StateMap = {
    "0": {
        display: "Unknown",
        colours: "bg-gray-100 text-gray-500 dark:bg-gray-100/10"
    },
    "1": {
        display: "Online",
        colours: "bg-green-100 text-green-500 dark:bg-green-100/10"
    },
    "2": {
        display: "Offline",
        colours: "bg-red-100 text-red-500 dark:bg-red-100/10"
    },

    "active": {
        display: "Active",
        colours: "bg-green-100 text-green-800"
    },
    "fallback": {
        display: "Fallback",
        colours: "bg-orange-200 text-orange-500 dark:bg-red-100/10"
    },
    "primary": {
        display: "Primary",
        colours: "bg-green-100 text-green-500 dark:bg-green-100/10"
    },
}

```

**Expected Behaviour**
![image](https://user-images.githubusercontent.com/51891628/154228162-21772806-b554-4266-93a6-47aeff5355d4.png)
![image](https://user-images.githubusercontent.com/51891628/154228127-919d2c3b-ee65-44e8-95c0-12baa81d2e20.png)
![image](https://user-images.githubusercontent.com/51891628/154228245-774c00c5-2c0b-44a0-9e1e-5d525cd5f74a.png)

**Current Behaviour**
![image](https://user-images.githubusercontent.com/51891628/154228392-121d5dd8-2391-414b-910e-9e9e2035eb67.png)
![image](https://user-images.githubusercontent.com/51891628/154228337-568f764a-ebce-4f28-89f9-722e83a9a5e6.png)
![image](https://user-images.githubusercontent.com/51891628/154228464-dfe08189-a94b-4f52-91a8-866ef8ce125d.png)

**Comment by adamwathan** [comment, marked as answer]
Is the state map file covered by your `content` configuration?

**Comment by imprisonedmind** [comment]
Thank you!

Silly of me for not thinking of that. 

To Fix: 
![image](https://user-images.githubusercontent.com/51891628/154259422-31be985e-3edb-4476-9550-e8404627a374.png)
